.rb

```diff
@@ -0,0 +1,47 @@
+puts "Welcome to the ADA Calculator"
+puts "Which operation would you like to use? "
+puts " 1. add(+)"
+puts " 2. substract(-)"
+puts " 3. multiply(*)"
+puts " 4. divide(/)"
+
+
+def get_operation
+  puts " So what is the operation you would like to use (type name or the symbol)?"
+  operation = gets.chomp
+  until operation == "+" || operation == "-" || operation == "/" || operation == "*"
```

Comment:
This is a good loop to "trap" user input, but you're not letting me enter "add" for example.